lightningd/peer_htlcs.c

```diff
@@ -721,7 +721,7 @@ static void htlc_accepted_hook_serialize(struct htlc_accepted_hook_payload *p,
 	}
 
 	json_add_hex_talarr(s, "next_onion", p->next_onion);
-	json_add_hex(s, "shared_secret", &hin->shared_secret, sizeof(hin->shared_secret));
+	json_add_hex(s, "shared_secret", hin->shared_secret, sizeof(struct secret));
```

Comment:
Gah, this is why writing helpers is important!

But yes, as @ZmnSCPxj points out, the commit msg is now nonsensical.

Reply:
So new `json_add_secret`?